docs/source/markdown/podman-build.1.md

```diff
@@ -416,7 +416,9 @@ Valid _mode_ values are:
 container full access to local system services such as D-bus and is therefore
 considered insecure.
 - **ns:**_path_: path to a network namespace to join.
-- **private**: create a new namespace for the container (default).
+- **private**: create a new namespace for the container (default). The
+**container** network mode is an alias for **private**, but has been deprecated
```

Comment:
Don't document it at all, just remove it.  It has been removed from buildah bud documentation for a while.  This was an over site.  We can still work with the option, but no need to mention it.

Reply:
Should I target this against the 2.2 branch instead, and we can completely remove in 3.0?

Reply:
I like the thought, but will let @rhatdan decide.

Reply:
Sure

Reply:
Updated PR to point at the v2.2 branch, which you can apparently do without recreating, which is kind of neat.

Reply:
Neat!  new learning for the day for me too!